debian

```diff
@@ -24,7 +24,7 @@ set -e
 
 start() {
   printf "Starting '$NAME'... "
-  start-stop-daemon --start --chuid "$USER:$GROUP" --background --make-pidfile --pidfile /var/run/$NAME.pid --exec "$APPDIR/$APPBIN" -- $APPARGS || true
+  start-stop-daemon --start --chuid "$USER:$GROUP" --background --make-pidfile --pidfile /var/run/$NAME.pid --chdir "APPDIR" --exec "$APPBIN" -- $APPARGS || true
```

Comment:
Yes somehow i got this right in my local script, but not here. It should definitely be `--chdir "$APPDIR"`